### ISSUE TITLE
Forge-py research

### ISSUE THREAD

**Issue by juliacollins** [nsidc/granule-metgen#5]
Add links to forge-py evaluation, placeholder area in software architecture diagram list, any other pre-project activities.

Relevant repository is [https://github.com/nsidc/forge-py](https://github.com/nsidc/forge-py), branch is `granule-metgen-investigation`

**Comment by lisakaser** [comment]
@juliacollins our original statement of the forge-py research phase in the research project stakeholder meeting notes was stated as
"Confirm usability of forge-py as a replacement for MetGen spatial handling. Ensure spatial handling can be executed independent of the rest of the ingest process."
This is what I would have expected to see addressed in this issue. Can you summarize your findings on this here or point me to an other issue that addresses that?

**Comment by juliacollins** [comment]
`forge-py` as-is can't be used as a drop-in tool, since it assumes all spatial information is read from a netCDF data file, and also generates a convex hull rather than a concave one. However, it does provide a reference to other packages that we _can_ use for spatial refinements, and I was able to hack some of the configuration values to generate a spatial output that roughly mimicked our current approach. Based on my experiments, I recommend we develop our own standalone command-line tool that accepts spatial information in a standalone input file, similar to the spatial input MetGen currently requires. This bit of development should be small (e.g. a sprint or less for a developer) since we're not starting from scratch. This code should be something that can either run on local infrastructure or wrapped in a lambda for use in a Cumulus pipeline.

Before we embark on any custom code, I do recommend reaching out to PO.DAAC to confirm whether or not they're interested in some of the changes we need. If so, then we could consider modifying their code base rather than creating one of our own. My experiments were quick-and-dirty and not elegant with respect to the existing code, and the logic would need some cleanup and better error handling.

Experiment notes are in the README in the branch `granule-metgen-investigation` for `forge-py` (https://github.com/nsidc/forge-py). This repository/branch can serve as the starting point for our "spatial handling" work.

**Comment by lisakaser** [comment]
@juliacollins thanks for the summary. I created a follow up issue (#22 ) to contact PO.DAAC.
